gulpfile.babel.js

```diff
@@ -113,3 +113,13 @@ gulp.task('assets', ['assets:main', 'assets:vendor']);
 /* Building all frontend assets */
 gulp.task('build', ['assets', 'css', 'fonts', 'js']);
 /* End of building all frontend assets */
+
+
+/* Watcher */
```

Comment:
Could You please fetch `master` tip from `komitywa` and merge it to Your branch - this diff looks really strange, because `gulp.task('watch...` is already in `gulpfile.babel.json` and it looks like You're putting it there in this pull request.